conf/test_full.config

```diff
@@ -14,14 +14,13 @@ params {
     config_profile_name        = 'Full test profile'
     config_profile_description = 'Full test dataset to check pipeline function'
     // Input data for full size test
-    // TODO nf-core: Specify the paths to your full test data ( on nf-core/test-datasets or directly in repositories, e.g. SRA)
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input               = "${projectDir}/assets/samplesheet.csv"
+    input               = "https://raw.githubusercontent.com/nf-core/test-datasets/detaxizer/samplesheets/samplesheet.full.csv"
 
     // Genome references
     fasta               = "s3://ngi-igenomes/igenomes/Homo_sapiens/NCBI/GRCh38/Sequence/WholeGenomeFasta/genome.fa"
+
     // Kraken2 test db
-    kraken2db           = "https://genome-idx.s3.amazonaws.com/kraken/k2_standard_08gb_20231009.tar.gz"
+    kraken2db           = "https://zenodo.org/records/8339732/files/k2_HPRC_20230810.tar.gz"
```

Comment:
Is that file large? If yes, that might be problematic. Using the S3 files is more reliable with test_full.